docs/Xamarin.Forms.Core/ns-Xamarin.Forms.Core.xml

```diff
@@ -0,0 +1,6 @@
+<Namespace Name="Xamarin.Forms.Core">
```

Comment:
i think we can drop this

Reply:
You mean the file itself ('docs/Xamarin.Forms.Core/ns-Xamarin.Forms.Core.xml') or the namespace tag? It was auto-generated when I ran docs.

Reply:
yeah the file itsef